packages/http/src/util.js

```diff
@@ -76,24 +76,27 @@ export function request(method, path, params, callback = s => s) {
       key => key.toLowerCase() === 'content-type'
     );
 
-    const hasContentType = !!contentTypeKey;
-    const contentTypeValue = hasContentType ? headers[contentTypeKey] : '';
+    const hasContentType = contentTypeKey && headers[contentTypeKey];
```

Comment:
But `contentTypeKey` is only exists in the user's headers? So this test is not needed